### ISSUE TITLE
[DESIGN] My VA Dashboard: Revisit Direction/Wireframe explorations (mobile & desktop)

### ISSUE THREAD

**Issue by mshea0606** [department-of-veterans-affairs/va.gov-team#15115]
## Background
We are taking this sprint (S33) to revisit the direction of the MyVA dashboard redesign and explore different options and decide upon the approach going forward. The focus should be on sketches/low-fi wireframes not on detailed visual design.

My VA goals: Get users to personalized information and tasks as quickly as possible. The page should be personalized, actionable, and scalable.

[Mobile wireframes](https://cassandraallen738782.invisionapp.com/console/HP-mobile-logged-in-ckeyhydog01kz01xd3tao5pcj/ckgkztfzz09ib01ea4mv33wh0/play)

### Mobile wireframes

- [x] Wireframe/sketch explorations
- [x] Regularly share updates in Github
- [x] Meet with Samara/Ryan/Matt as needed
- [x] Decide on direction going forward

### Desktop wireframes

- [ ] Convert mobile designs into desktop mockups

**Comment by tressaellen** [comment]
@callen2563 @Samara-Strauss @rtwell @mshea0606 @sshein - here is the first dump of wireframe directions. For this first push we explored the amount of information shown to a user. I'll be posting each in a separate comment so it's a little easier to track feedback. 

This first one is the 'x-small' version aka springboard approach. Basically the very very minimum amount of information that we could show to still get users to their tasks. Basically elevating general status updates that would link straight to each specific page. Then below would be links to their specific pages as well but essentially no specific information on their dashboard, hence the 'springboard approach'

![myVA 1 (xsm).jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/711edd84-d690-4759-998e-81aed57291ad)

Next up we have the 'small' version - non-specific update notifications but with a little bit more specific information in the meat of the page.

![myVA 2 (sm).jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/1181eaa2-2331-41d7-8360-bc582e173e2f)

**Comment by Samara-Strauss** [comment]
Awesome, thank you for sharing these! Looking forward to our discussion later.

**Comment by tressaellen** [comment]
Third - we did a 'medium' version with a little bit more information included and where all notifications would be tied specifically to each collapsible section.

![myVA 3 (med)-closed.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/0c2f0ad9-8004-4c47-8508-96f87ec313f0)

We explored a couple different options for how to display these status updates/how much information to include. This first one has status updates separated into its own section up top (a la Samara's sketch).

![myVA 3 (med) - expanded 1.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/395d178c-680e-4179-91f0-5728633245f1)

This one includes more specific information, but the status updates are in each specific section instead of up top. The only specific information shown in each section would be action items, to see anything else the user would need to link to the specific page.  

![myVA 3 (med) - expanded 2.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/bf62d5bf-68c6-4243-a6e9-d6b03e79d79b)

The last one in the 'med' category - same idea as the previous iteration, but with parsed down information in the status updates.

![myVA 3 (med) - expanded 3.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/81190919-dc5f-47ee-a5ef-c9d093ffdf77)

**Comment by tressaellen** [comment]
Lastly we have our 'large' category - basically any and all information included as well as a couple more ideas on how to show status updates. 

This one is basically elevating all status notifications but without specific information.

![myVA 4 (lrg) - status 1.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/b2a1b03f-00a0-49cc-b15e-cf49ba65a014)

This one has all status updates in its own dropdown aka update center. 

![myVA 4 (lrg) - status 2.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/770dc33c-05b2-4fa7-95af-0dcabbf67302)

![myVA 4 (lrg) - status 2 open.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/c5552e73-dac5-40c8-b756-3fdd2af9353c)

And then we have the fully expanded, everything possible to show.

![myVA 4 (lrg) -expanded.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/853c6997-3240-4177-a20c-904ee50162f5)

**Comment by tressaellen** [comment]
I've added [all the screens to invision](https://cassandraallen738782.invisionapp.com/console/HP-mobile-logged-in-ckeyhydog01kz01xd3tao5pcj/ckgkztfzz09ib01ea4mv33wh0/play) as well in case that's a little bit easier to peruse.

**Comment by Samara-Strauss** [comment]
This is really awesome work. See you at 2:00 ET!

**Comment by Samara-Strauss** [comment]
Here are my notes from our meeting earlier. From what I heard, people saw the most potential in the Small option and Medium, Option 2.

**Next Steps: @tressaellen and @callen2563 want to try a couple more options that incorporate the pro/con feedback discussed in the meeting.

---------------

## X-small

**Goal**: What is the absolute minimum we could show a user?

**Pros**

- Really simple and easy.
- Users would be able to figure out what they need to do quickly.
- On mobile, there wouldn’t be much need for scrolling.
- “A true dashboard.”
- Summary — “You have 5 new updates” —> You know what you’re working with without scrolling.
- Having the updates at the top.

**Cons**

- A user can’t get specifics from this page.
- Does not give us enough for us to feel like this page is more valuable than the homepage.

## Small

**Goal**: How can we take a simplistic approach but be a little bit more personalized?

**Pros** 

- A little more info than totally bare bones.
- More robust.
- Really easy to understand.
- Gives the user a good general idea of their status at the VA.
- Not a lot of scrolling (potentially take a simpler approach on mobile and show more on desktop).
- Gives more value than the homepage because there is more personalized information.

**Cons**

- Is there a purpose to have an update section AND the overview section? Feels weird to separate out this information.

## Medium, Option 1

**Goal**: Display more detailed updates before the “overview” section. Has same treatment for the overview section as the “Small.”

**Pros**

- More detail in the “updates” section.
- Has a lot more value than the homepage.
- Good balance between helpful, actionable information and scalability.

**Cons**

- The “updates” section would take up a lot of room if you had a lot, and potentially something could be easily missed.
- This page is missing a title.

**Questions**

- Should we separate updates by section?

## Medium, Option 2

**Goal**: Show status updates within each section.

**Pros**

- “Status” number shows how many updates in a section.
- Only shows changes, not general things (eg. Might show a canceled appointment but not upcoming).
- Only elevates actionable items.
- Collapsible sections allow you to easily scan for updates on mobile.
- Categories help the page feel more organized.
- A lot more info than the homepage, so page has more value.

**Cons**

- There may be some scalability issues as the page grows.

**Questions**

- Should we show collapsible information on mobile?
- Do we show “updated” sections defaulted as open? Or bump them to the top?

 ##Medium, Option 3

**Goal**: Showing same info as #2 but condensing down updates to a simpler format.

**Pros**

- Super scannable; Might not need to be collapsible on mobile.

**Cons**

- Not specific enough on claim information. The user shouldn’t have to click to know what that’s talking about.

**General observation**

- There’s a careful balance between too much information and not enough.

## Large

**Goal**: All the things! Updates are at the top with the full dashboard below.

**Pros**

- Updates are at the top so you can see them immediately.

**Cons**

- If we took a dropdown approach with the updates, someone might miss those.
- Is it confusing to show updates at the top AND update numbers next to the sections themselves?
- Feels like there is a lot of dropdown on the page.

**Comment by mshea0606** [comment]
Thanks @Samara-Strauss for adding in these notes!

**Comment by callen2563** [comment]
@Samara-Strauss Hi Samara, we would love to get your thoughts on the 3 variations based on Medium 2 wireframe..

[myVA 3 (med) - V2 A](https://user-images.githubusercontent.com/68560244/97050282-f1d85700-1531-11eb-977f-cb4a1f3273d9.jpg)
![myVA 3 (med) - V2 C](https://user-images.githubusercontent.com/68560244/97050286-f43ab100-1531-11eb-834e-1e0a68f8eb39.jpg)
![myVA 3 (med) - V2 B](https://user-images.githubusercontent.com/68560244/97050287-f56bde00-1531-11eb-8c91-64abce3287eb.jpg)

**Comment by Samara-Strauss** [comment]
### Wireframe 1

https://user-images.githubusercontent.com/68560244/97050581-77f49d80-1532-11eb-8c59-60576674ddb5.jpg

- This needs a page title.
- Without a section title for the updates, I worry that they'll feel like they're floating without context on desktop.
- I miss the overview information we had included in (I think) the "small" option. Without it, this page doesn't feel like it carries enough weight to route people here instead of the homepage. 
- I think elevating the updates to the top works better for people who only might have one or two (or no) updates. 

### Wireframes 2 and 3

https://user-images.githubusercontent.com/68560244/97050568-74611680-1532-11eb-9513-cf30f18f812d.jpg
https://user-images.githubusercontent.com/68560244/97050580-76c37080-1532-11eb-91d0-f907823de540.jpg

Since these were similar, I'll give feedback on both together.

- Overall, I like the balance of information here more, and this is the design direction I personally lean towards. That being said, how could we make this useful for someone who had no information? I think this has promise for people with a lot of updates and people with a medium to small amount (which I think is what we are looking at in the mockup – someone with 3 things). But I worry that there would be nothing to do for people who don't have anything. Maybe that's something to think about for next week.
- Question: What is in the My VA menu? If all the sections are listed out on the main page (which it looks like they are), then why do we need a menu? 
- In terms of where we show the notification numbers, I think the treatment in wireframe 2 works better. However, I can already tell that it would be an accessibility issue, so I would do one of the following:
  - Create a label state that says "2 updates" or something like that.
  - Have a sentence below the section heading that says "You have 2 updates." Perhaps the number is stylized in a circle or something to call it out to sighted user, but the pattern would still work or a person who uses a screen reader.
- For both designs, I am genuinely unsure whether we could make the calls to action dynamic re: how many other prescriptions/claims/messages/appointments someone has. I would ask Erik and Sandra about this, but I think the answer might be this isn't possible (or it is but is annoying to code). We could either not include the total number, or we could incorporate the total number another way.
- Oh, also, for the "Apply for benefits" section, there's no such thing as a "start an application" CTA. All of the applications are on different pages, so there would be nowhere to link to for a singular CTA. So, we should combine Explore benefits into the Apply for benefits section. That section should be about continuing existing apps _and_ exploring new benefits.

**Comment by Samara-Strauss** [comment]
I posted this on Slack, but also wanted to post here — I think the next steps here are to work through these iterations for:

- Someone who has no benefits and is completely new.
- Someone who has even more stuff that what is shown above. **This part is different than what I posted in Slack** — here are three **real** new features that are in progress. We do NOT have to consider these in the MVP but we WILL have to consider how to elevate them later on. How would the dashboard work with the following?:
  - The eBenefits team is adding payment history to VA.gov. How would the dashboard work if we showed someone's most recent payment and linked to this tool?
  - The caregiver team is figuring out how to give patients access to medical forms before their an appointment so patients can fill these out before going to their VA doctor. How would we surface these?
  - The BAM2 team is working on a tool that will let veterans know if they owe a debt to the VA (unfortunately, this can happen if the VA erroneously overpays someone). How would we help elevate that someone owes the money to the VA?

I think working through these scenarios will help us decide on our final direction. Let me know if you have any questions!

**Comment by tressaellen** [comment]
From the latest meeting:

Now that we have two solid directions we'd like to move towards testing these versions to see what information veterans are looking for. 

Version A (more simplified):

![myVA 3 (med) V3 - A.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/3c98cec7-3aa4-4475-95f0-684e1bd1224b)

Version B (more information):

![myVA 3 (med) - V3 B collapsed.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/440509df-34fb-4b73-8dd7-f9e46559a842)

![myVA 3 (med) - V3 B.jpg](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/f0ccbcea-c811-495d-be68-4bfac4539d90)

**Comment by Samara-Strauss** [comment]
@tressaellen @callen2563 when you have your latest mobile updates done, can you post them here?

Also, I updated this ticket to account for the desktop wireframes as well. Looking forward to see the adaptations!

**Comment by tressaellen** [comment]
Here are the latest updates.

![myVA C1](https://user-images.githubusercontent.com/58752772/97627535-5f77fd80-19e0-11eb-9112-dfa017572485.jpg)
![myVA C2](https://user-images.githubusercontent.com/58752772/97627540-60109400-19e0-11eb-9157-7e280e8dea14.jpg)
![myVA D1](https://user-images.githubusercontent.com/58752772/97627542-60109400-19e0-11eb-8b2e-4d90ee4104ca.jpg)
![myVA D2](https://user-images.githubusercontent.com/58752772/97627543-60a92a80-19e0-11eb-95e1-bf514e00ffc0.jpg)

**Comment by callen2563** [comment]
Here are the version C desktop wires to go with the mobile screens. We've added the accordion menu from mobile as a side nav on desktop in this version. We also have a scrolling page, non side nav version to be posted in the am.

![Healthcare VA Desktop C](https://user-images.githubusercontent.com/68560244/97923349-b0566180-1d12-11eb-918a-4ad4583a61b7.jpg)
![Claims   Appeals VA Desktop C](https://user-images.githubusercontent.com/68560244/97923360-b3515200-1d12-11eb-9540-ab5350481e7c.jpg)

**Comment by Samara-Strauss** [comment]
Hey — thank you for this! Tbh, I was surprised to see the tabbed layout — I thought the desktop view would be a single page dashboard that was pretty analogous to the mobile mockups linked right above this. The above desktop mock feels like we totally went back to the original tabbed/info heavy approach we were talking about a couple sprints ago. What happened to the dedicated update section (looks like it's now split across sections)? What happened to the "overview" info (eg. 3 new messages, etc)? I know we can technically show more info on the desktop view than the mobile view, but these feel too far apart, and I think that would both be confusing to users and also weird in terms of build (how do we make it responsive)? 

TL; DR — I think we have the right balance of info in the mobile view. Can we see a single-page desktop adaptation that is more analogous to the mobile mockups we've been working on? Hint: we don't need to add a ton more information, at least not at this point in our explorations.

Let me know if you have questions. Thanks!

**Comment by callen2563** [comment]
@Samara-Strauss @tressaellen 
<img width="865" alt="Screen Shot 2020-11-02 at 5 42 52 PM" src="https://user-images.githubusercontent.com/68560244/97937333-3da9ae00-1d33-11eb-9c89-1fc8a1cb60c8.png">

So the thought for version C was that the mobile accordion would respond and become the nav on desktop so that the user could see their updates in each section instead of having to scroll to see what the updates are in each section. We also are in the process of mocking up the scrolling desktop to go with mobile version A / D that show the notifications at the top of the page.

**Comment by Samara-Strauss** [comment]
Ah, I see. I would talk to Sandra and Erik about whether something could work this way responsively. In the mobile view, the collapsed elements are the whole section, but on desktop, you've separated the section headers into a side navigation. For responsive design, ideally, the elements are a 1:1 analogy. So, if we had a side nav on desktop, that would collapse into a navigation on mobile. Instead, we have section headers on mobile that have morphed into navigation on desktop. While that probably is possible to code, it would be wonky because we're fundamentally changing what these elements are across viewports.

Personally, I would still like to see the single page, no side-nav version of [this mobile design](https://user-images.githubusercontent.com/58752772/97627540-60109400-19e0-11eb-9157-7e280e8dea14.jpg). Forget the old health care mockups for a second. How would you convert this, with no new elements or data, to a desktop layout?

**Comment by tressaellen** [comment]
@rtwell @Samara-Strauss @callen2563 

Okay, here's a few more mocks to look at, including the long scroll design from mobile design D. First is the updated mock for mobile design C (seen below), with mobile navigation (for conversations sake).

![Claims   Appeals VA Desktop C2 1](https://user-images.githubusercontent.com/58752772/97999158-70858d80-1d97-11eb-86f6-d3711a812f3e.jpg)
![Healthcare VA Desktop C2](https://user-images.githubusercontent.com/58752772/97999160-711e2400-1d97-11eb-86fc-98c41843df48.jpg)

Next is the collapsed version of mobile C (1:1 to the mobile version)

![Healthcare VA Desktop C3 1](https://user-images.githubusercontent.com/58752772/98001001-99a71d80-1d99-11eb-9031-9f03a026f6c0.jpg)
![Healthcare VA Desktop C3](https://user-images.githubusercontent.com/58752772/98001004-9b70e100-1d99-11eb-892f-ab6dbdb0ef4a.jpg)

And then finally desktop version of mobile D.

![MyVA D Desktop1](https://user-images.githubusercontent.com/58752772/98001064-adeb1a80-1d99-11eb-897e-24d322a0c4da.jpg)
![MyVA D Desktop2](https://user-images.githubusercontent.com/58752772/98001070-afb4de00-1d99-11eb-9d6f-0ba9ae36f4b0.jpg)

**Comment by Samara-Strauss** [comment]
I think we need to meet to talk about these:

- The side nav issue is not an issue of whether I think the side nav is a good idea or not. It's about creating something that would be complex to build (as far as I know) because you've changed what an element is used for on mobile (a section heading) and made it a fundamentally different element that will function differently on desktop (a side navigation). I would talk to Sandra and Erik about how this might work from a technical perspective to see if something like this is practical from a build perspective.
- I don't think we need the collapsible elements on desktop because we would never show sections collapsed on desktop. Let's remove the divider lines both on desktop and mobile (it feels super clunky on desktop), and only have the option to collapse/expand on mobile (AFAIK, this should be OK to do responsively).
- I am still seeing iconography and data elements in [this desktop design](https://user-images.githubusercontent.com/58752772/98001001-99a71d80-1d99-11eb-9031-9f03a026f6c0.jpg) that I'm not seeing in [this mobile design](https://user-images.githubusercontent.com/58752772/97627540-60109400-19e0-11eb-9157-7e280e8dea14.jpg). Can you clean that up? If you're going to show icons on desktop, show them on mobile. If you're going to show the old health care header on desktop (that includes "Learn more about your health care benefits), show that on mobile. 

Overall, tbh, all of the desktop views feel really clunky to me, and I don't feel like any of them are quite there yet. However, unless you were set on presenting the desktop view in testing, we could potentially clean up some of the things I mentioned above and continue to finesse these as we go along. I, personally, don't think we have to present the desktop view in testing, so I don't want us to sink a bunch of time into making these perfect, especially as things still might change based on our testing findings. Let me know how you'd like to proceed from here.

**Comment by tressaellen** [comment]
Let's meet to make sure we're on the same page for these... I'd like to get to a better spot with desktop before testing, just so we at least have an idea of what we're working with even though things might change with testing. I'll slack you and figure out a time to chat.

**Comment by Samara-Strauss** [comment]
Short summary notes from yesterday's regroup with Tressa and Ryan:

- The desktop view might be different presentationally, but it should **not** be different in terms of data/info we show **unless** there are data-backed, well-rationaled reasons for these differences (example: if we end up showing collapse/open functionality on mobile, we don't need to show that on desktop because there is enough real estate that we don't need to collapse sections).
- We do *not* need to show both mobile and desktop mockups in testing since they are going to be the same in terms of data/info.
- When we get to the desktop view, we need to figure out how to make the design not feel super clunky. This might mean figuring out some layout/presentational elements on desktop and then adapting the responsive mobile mockups accordingly. Essentially, both designs will need to evolve together since they're just responsive views of one another.
- Otherwise, we are good to move forward with testing prep, and we can close this ticket! Congratulations on a great explorations sprint!

**Comment by sshein** [comment]
just want to say if sections are collapsed on mobile and expanded on desktop, that will be a pretty different experience for users and could be worth testing both, even if the data is ultimately the same.
